app/javascript/packs/outdoor_map_builder.js

```diff
@@ -162,6 +162,115 @@ console.log('[MAP] Layers built');
 
 // mapbox token
```

Comment:
I know that it's not your code, but if you decide to change anything it would be great if you could remove this comment^^